core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -58,6 +62,46 @@ var StoryEditorPage = function() {
   var unpublishStoryButton = element(
     by.css('.protractor-test-unpublish-story-button'));
 
+  /*
+   * CHAPTER
+   */
+  var initialChapterSelect = element(
+    by.css('.protractor-test-initial-chapter-select'));
+  var explorationIdInput = element(
+    by.css('.protractor-test-exploration-id-input'));
+  var explorationIdSaveButton = element(
+    by.css('.protractor-test-exploration-id-save-button'));
+  var nodeOutlineEditor = element(
+    by.css('.protractor-test-add-chapter-outline'));
+  var nodeOutlineSaveButton = element(
+    by.css('.protractor-test-node-outline-save-button'));
+  var addPrerequisitesSkillButton = element(
```

Comment:
Typo? Should this be `addPrerequisiteSkillButton`?

Reply:
Updated